objects/_objects.layout.scss

```
@@ -87,12 +87,57 @@ $inuit-layout-huge-gutter:  inuit-spacing(huge) !default;
    * 2. By default, all layout items are full-width (mobile first).
    * 3. Gutters provided by left padding:
    *    http://csswizardry.com/2011/08/building-better-grid-systems/
+   * 4. Set a default order, so we can increase of decrease that number with
```

suggestion: Correct typo
```suggestion
   * 4. Set a default order, so we can increase or decrease that number with
```